content/guides/golang/quick-start.md

```diff
@@ -101,7 +101,7 @@ defer childSpan.finish()
 
 ## Make an HTTP request
 
-To get traces across service boundaries, we propgagte [context](https://golang.org/pkg/context/) by injecting the context into http headers. Once the downstream service recieves the http request, it must extract the context and continue the trace. (The code example doesn't handle errors correctly, please don't do this in production code; this is just an example)
+To get traces across service boundaries, we propogate [context](https://golang.org/pkg/context/) by injecting the context into http headers. Once the downstream service recieves the http request, it must extract the context and continue the trace. (The code example doesn't handle errors correctly, please don't do this in production code; this is just an example)
```

Comment:
```suggestion
To get traces across service boundaries, we propagate [context](https://golang.org/pkg/context/) by injecting the context into http headers. Once the downstream service receives the http request, it must extract the context and continue the trace. (The code example doesn't handle errors correctly, please don't do this in production code; this is just an example)
```

Reply:
Did I really do that? Silly me... thank you!